soc/arm/nordic_nrf/nrf51/Kconfig.defconfig.series

```diff
@@ -17,6 +17,13 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 	int
 	default 32768
 
+config SYS_CLOCK_TICKS_PER_SEC
```

Comment:
Would it make sense to move this definition one level up, so we don't get to write it three times?
One can always overwrite it in project...

Reply:
Done; thanks.